File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int n = nums.length;
        int count=0;
        for(int val:nums){
            int dig = 0;
            while(val>0){
                dig++;
                val/=10;
            }
            if(dig%2 == 0){
                count++;
            }
        }
        return count;
    }
}

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        n = len(nums)
        count = 0

        for i in range(n):
            dig = 0
            while(nums[i] > 0):
                nums[i]//=10
                dig+=1
            if(dig%2 == 0):
                count+=1
        return count